packaging/rpm/drlm.spec

```diff
@@ -144,9 +144,15 @@ systemctl is-enabled --quiet drlm-gitd.service && systemctl disable drlm-gitd.se
 
 systemctl daemon-reload
 
-### Check if older versions than 2.4.12
 [ -f /usr/sbin/drlm ] && drlm_ver_num="$(awk 'BEGIN { FS="=" } /VERSION=/ { print $$2 }' /usr/sbin/drlm | awk -F. '{printf("%02d%02d%02d\n", $1, $2, $3)}')"
 if [ -n $drlm_ver_num ]; then
+  ### Check if older versions than 2.4.0
+  if [ $drlm_ver_num -lt 020400 ]; then 
```

Comment:
really where the functionality change is introduced and these variables will no longer be ignored is from this version so it should be 020412

Reply:
moved to 020412